src/main/java/bio/terra/cli/command/notebook/Create.java

```diff
@@ -0,0 +1,110 @@
+      description = "The Compute Engine machine type of this instance.")
+  private String machineType;
+
+  @CommandLine.Option(
+      names = "vm-image-project",
+      defaultValue = "deeplearning-platform-release",
+      description = "The ID of the Google Cloud project that this VM image belongs to.")
+  private String vmImageProject;
+
+  @CommandLine.Option(
+      names = "vm-image-family",
+      defaultValue = "tf-latest-gpu",
+      description =
+          "Use this VM image family to find the image; the newest image in this family will be used.")
+  private String vmImageFamily;
+
+  // TODO: Add boot disk types & gpu size configs.
+
+  @Override
+  public Integer call() {
+    GlobalContext globalContext = GlobalContext.readFromFile();
+    WorkspaceContext workspaceContext = WorkspaceContext.readFromFile();
+
+    AuthenticationManager authenticationManager =
+        new AuthenticationManager(globalContext, workspaceContext);
+    authenticationManager.loginTerraUser();
+    TerraUser user = globalContext.requireCurrentTerraUser();
+    String projectId = workspaceContext.getGoogleProject();
+
+    String command =
+        "gcloud beta notebooks instances create $INSTANCE_NAME "
+            + "--location=$LOCATION "
+            + "--instance-owners=$USER_EMAIL "
+            + "--service-account=$SERVICE_ACCOUNT "
+            + "--machine-type=$MACHINE_TYPE "
+            + "--vm-image-project=$VM_IMAGE_PROJECT "
+            + "--vm-image-family=$VM_IMAGE_FAMILY "
+            + "--network=$NETWORK "
+            + "--subnet=$SUBNET "
+            + "--metadata=^:^enable-oslogin=TRUE:installed-extensions=jupyterlab_bigquery-latest.tar.gz,jupyterlab_gcsfilebrowser-latest.tar.gz,jupyterlab_gcpscheduler-latest.tar.gz";
+    Map<String, String> envVars = new HashMap<>();
+    envVars.put("INSTANCE_NAME", instanceName);
+    envVars.put("LOCATION", location);
+    envVars.put("USER_EMAIL", user.terraUserName);
+    envVars.put("SERVICE_ACCOUNT", user.petSACredentials.getClientEmail());
+    envVars.put("MACHINE_TYPE", machineType);
+    envVars.put("VM_IMAGE_PROJECT", vmImageProject);
+    envVars.put("VM_IMAGE_FAMILY", vmImageFamily);
+    // 'network' is the name of the VPC network instance created by the Buffer Service.
+    // Instead of hard coding this, we could try to look up the name of the network on the project.
+    envVars.put(
+        "NETWORK", "projects/" + projectId + "/global/networks/network");
+    // Assume the zone is related to the location like 'us-west1' is to 'us-west1-b'.
+    String zone = location.substring(0, location.length() - 2);
+    // Like 'network', 'subnetwork is the name of the subnetwork created by the Buffer Service in
+    // each zone.
+    envVars.put(
+        "SUBNET",
+        "projects/"
+            + projectId
+            + "/regions/"
+            + zone
+            + "/subnetworks/subnetwork");
+
+    // TODO: Is there anyway to stream back the output?
```

Comment:
Not yet, but I have a ticket in this sprint to address this: https://broadworkbench.atlassian.net/browse/PF-424